ROADMAP.md

```diff
@@ -100,28 +104,222 @@ UPDATE:
 
 ##### Expected date of completion: `late August`
 
-# Milestone 3 - Enter IPLD - Enable js-ipfs to handle both merkledag protobufs and ipld cbor objects
+--------------------------------------------------------------------------------------------------
+
+# 2016 Q4
+
+# Milestone - The InterPlanetary JavaScript
+
+> Summary: Settle in what will be the 'good parts' for us to use accross JS projects
+
+### Leads:
+
+- David
+- Friedel
+- Victor
+- Richard
+
+### Tasks:
+
+- [ ] Create an issue for each of the 'parts' that needs to be discussed
+- [ ] Parts:
+  - [ ] 
+  - [ ] 
+
+### Dependencies:
+
+### Requirements by other projects: `NA`
+
+### Notes: `NA`
+
+##### Expected date of completion: `Week 1 - October 17 to create the issues, discussion will happen throughout the quarter`
+
+# Milestone - interface-floodsub
+
+> Summary: Define an interface for floodsub, that can be implemented for js-ipfs and js-ipfs-api
+
+### Leads:
+
+- David
+- Juan
+
+### Tasks:
+
+- [ ] Create interface-floodsub.md on pubsub-notes repo
+
+### Dependencies:
+
+### Requirements by other projects:
+
+This is a requirement to integrate floodsub in js-ipfs
+
+### Notes: `NA`
+
+##### Expected date of completion: `Week 1 - October 17`
+
+
+# Milestone - Awesome IPLD - Enable js-ipfs to handle both merkledag protobufs and ipld cbor objects
 
 > Summary: Integration of IPLD (js-ipld and js-ipld-ipfs) into js-ipfs.
 
+### Leads:
+
+- David
+
 ### Tasks:
 
-NA
+- [ ] Awesome IPLD endeavour PR -
 
 ### Dependencies:
 
-- [ ] IPLD migration path
+- [x] CID spec
 - [ ] go-ipfs with IPLD already integrated (it would be best)
 
 ### Requirements by other projects:
 
+- everyone is waiting for this
+
+### Notes:
+
+NA
+
+##### Expected date of completion: `Week 2 - Oct 24`
+
+
+
+# Milestone - Async Crypto
```

Comment:
@dignifiedquire can you complete this milestone with the tasks that still need to happen for it to be complete?